central/centralhealth/service/service_impl.go

```diff
@@ -46,6 +49,32 @@ func (s *serviceImpl) AuthFuncOverride(ctx context.Context, fullMethodName strin
 
 // GetUpgradeStatus returns the upgrade status for Central.
 func (s *serviceImpl) GetUpgradeStatus(ctx context.Context, empty *v1.Empty) (*v1.GetUpgradeStatusResponse, error) {
+	// TODO: Need to figure out if there is anything we can do with size in Postgres as that data
+	// IS NOT accessible from central
+	if features.PostgresDatastore.Enabled() {
+		upgradeStatus := &v1.CentralUpgradeStatus{
+			Version: version.GetMainVersion(),
+			//CanRollbackAfterUpgrade:               int64(freeBytes)+toBeFreedBytes > requiredBytes,
```

Comment:
We do not have access to the capacity of postgres.  That is only available from the file system where the database resides.  I'm not sure what to do with this.  Thoughts?

Reply:
I can, BUT that will not work if we use a managed service for the database.  In those instances we will have to query the provider.  I think I'm going to write another subtask to cover this and note that in the TODO unless you would prefer to tackle it with this one.

Reply:
ROX-12059

Reply:
👍 we can also potentially pass the max to Central

Reply:
@connorgorman are you talking about setting something like an env var for central based off the central-db pvc size config in values?  Or just setting an env var?  Something else?  Lots of permutations on what the max size could be.  With a pvc it is configurable.  It isn't really with a hostPath.  It is something else entirely when we standup a managed service with a managed Postgres, though I assume that value could be a parameter.  And the max is "infinite" if we back it with Aurora Serverless.

Reply:
Yeah, I think there are definitely some complications with that